engine/install/debian.md

```diff
@@ -31,16 +31,23 @@ architectures.
 
 ### Uninstall old versions
 
-Older versions of Docker went by the names of `docker`, `docker.io`, or
-`docker-engine`, you might also have installations of `containerd` or `runc`.
-Uninstall any such older versions before attempting to install
-a new version:
+Older versions of Docker went by the names of `docker`, `docker.io`, and
+`docker-engine`. You must make sure that such old versions of Docker aren't
+installed on your system before you install a current version.
+
+Docker Engine also depends on `containerd` or `runc`. Likewise, if you have
+those packages installed previously, you may want to uninstall them to avoid
+conflicts.
+
+Run the following command to uninstall old versions of Docker Engine,
+`containerd`, and `runc`.
 
 ```console
-$ sudo apt-get remove docker docker-engine docker.io containerd runc
+$ for pkg in docker docker-engine docker.io containerd runc; do sudo apt-get remove $pkg; done
 ```
 
 `apt-get` might report that you have none of these packages installed.
+That's fine.
```

Comment:
```suggestion
```

Not sure that's necessary, but in these cases I've never been sure what else to use… Possibly just nothing is best